website/docs/language/state/encryption.mdx

```diff
@@ -161,7 +169,8 @@ The PBKDF2 key provider allows you to use a long passphrase as to generate a key
 
 | Option                   | Description                                                                                                                                             | Min.      | Default                            |
 |--------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|------------------------------------|
-| passphrase *(required)*  | Enter a long and complex passphrase.                                                                                                                    | 16 chars. | -                                  |
+| passphrase *(required)*  | Enter a long and complex passphrase. Required if `chain` is not specified.                                                                              | 16 chars. | -                                  |
+| chain *(required)*       | Receive the passphrase from another key provider. Required if `passphrase` is not specified.                                                            |           | -                                  |

```

Comment:
Just a note for other readers, who might not be too familiar with the encryption implementation.

I was a bit surprised by not finding any code which makes it possible to evaluate chains (this PR doesn't touch anything outside of specific key providers). Then I realized we already have it in place but with no actual user-facing features.

`targetBuilder.setupKeyProviders` is what makes it possible so if anyone else got confused, reading through that function makes it more clear.

Also, I am wondering, should we put more tests into that chaining feature? No one used it before and we may introduce something unexpected by enabling this. WDYT?

Reply:
Is it possible to use chaining with non-experimental key providers?

Reply:
Indeed, it doesn't seem to be much of a risk. I am good, thanks!